docs/advanced/navigating-politeia-data.md

```diff
@@ -2,101 +2,83 @@
+| **signature**              |    |  
+| **receipt**             |   |  
 
-**comments.journal**
+### comments.journal
 
-This file contains two types of row
+The `comments.journal` file contains data related to comments. The table below describes the parameters recorded for each comment.
 
-**action:add** rows describe the addition of comments.
 
-* **token** is the proposal being commented on
-* **parentid** is either 0 for top-level comments, or the comment_id of the comment which is being responded to
-* **commentid** is an identifier for each comment
-* **comment** is the body (text) of the comment
-* **timestamp** is a unix timestamp for when the comment was made
-* **publickey** is the public key of the Politeia user who made the comment
-* **censored** records whether the comment has been censored, where censored == true the comment will not be shown on proposals.decred.org
-* **signature** 
-* **receipt**
-* **totalvotes**
-* **resultvotes**
+| Parameter           | Description                   |  
+| -------------------- | ---------------------- | 
+| **action**             | The action taken by the commenter. If a comment is added, `action` = add. If a comment is voted on, `action` = addlike. If `action` = addlike, the comment action is additionally assigned '1' for upvote or '-1' for downvote.    |  
+| **token**             | The proposal being commented on.  |  
+| **parentid**             | The id of the parent comment, if a parent exists. If no parent comment exists (i.e. top-level comment), `parentid` = 0.  |  
+| **commentid**              |  The identifier (id) for the comment  |  
+| **timestamp**              |  A unix timestamp specifying time comment was submitted |  
+| **publickey**             |  The public key of the Politeia user who made the comment |  
+| **censored**             |  The censorship status. If the comment has been censored, `censored` = true, and the comment will not be shown on proposals.decred.org.  |  
+| **receipt**             |   |  
+| **totalvotes**            |   |  
+| **resultvotes**             |   |  
 
-**action:addLike** rows describe the addition of up/down votes on comments, many of the fields have the same meaning as for action:add rows. 
 
-* **action** can be either 1 (upvote) or -1 (downvote)
+### Identifying users
 
-In these data-sets, **Politeia users are identified by their public keys**. The method of associating a public key with a username is a little convoluted at present, in that one must start with the username. There is currently no way to look up a username from a pubkey, but that feature will be added.
+In these datasets, **Politeia users are identified by their public keys**. The method of associating a public key with a username is a little convoluted at present, in that one must start with the username. There is currently no way to look up a username from a pubkey, but that feature is planned. 
```

Comment:
Why are we emphasizing "Politeia users are identified by their public keys"?

Also should we remove the part about being convoluted? We can just say what is currently possible and what is planned for the future.

Reply:
Agreed. Have created 'User data' section, and removed wording about 'convoluted' and 'future plans' (I find it's always a bad idea to commit to future features in documentation).